docs/explanations/modeling_extensions/surrogate/api/omlt-keras/omlt-keras-options.rst

```diff
@@ -0,0 +1,82 @@
+keras_surrogate Options
```

Comment:
This looks good to me, a more experienced OMLT user may have further suggestions.